### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by mattip**
- Add blurb about PyPy
- Change some examples of python use to python3.9
- Soften the commitment about latest Python versions
- Mention the commercial nature of the default channel

The last point could use some careful review: I am not sure I got the content and tone right.

### CODE REVIEW

**Comment by mattip** [comment]
Any thoughts on this PR?

**Comment by cjmartian** [comment]
@anaconda-issue-bot check

**Comment by jezdez** [comment]
@chenghlee Could you take a closer look at the documentation changes here if they match what Anaconda's policy is at the moment?

**Review by chenghlee — CHANGES_REQUESTED** [review]

File: docs/source/user-guide/tasks/manage-channels.rst [hunk 1]
```diff
@@ -8,7 +8,8 @@ Conda packages are downloaded from remote channels, which are URLs to
 directories containing conda packages. The conda command searches a default
 set of channels and packages are automatically downloaded and updated
 from https://repo.anaconda.com/pkgs/. Read more about
-:doc:`conda channels <../concepts/channels>`.
+:doc:`conda channels <../concepts/channels>`. Note that this default channel
+may require a license for commercial use.
```
[the remark this answers is not in the document]
Comment (mattip) [reply docs/source/user-guide/tasks/manage-channels.rst:12]: changed to "and the various terms of service for their use"

File: docs/source/user-guide/install/download.rst [hunk 2]
```diff
@@ -9,7 +9,8 @@ Downloading conda
 
 You have 3 conda download options:
 
-* `Download Anaconda <https://www.anaconda.com/download/>`_---free.
+* `Download Anaconda <https://www.anaconda.com/download/>`_---free for
+  non-commercial use.
```
[the remark this answers is not in the document]
Comment (mattip) [reply docs/source/user-guide/install/download.rst]: adopted

File: docs/source/user-guide/concepts/channels.rst [hunk 3]
```diff
@@ -16,9 +16,10 @@ They serve as the base for hosting and managing packages.
 Conda :doc:`packages <../concepts/packages>` are downloaded
 from remote channels, which are URLs to directories
 containing conda packages.
-The ``conda`` command searches a default set of channels,
-and packages are automatically downloaded and updated from
-https://repo.anaconda.com/pkgs/.
+The ``conda`` command searches a set of channels. By default,
+packages are automatically downloaded and updated from
+the ``default`` channel https://repo.anaconda.com/pkgs/ which may require
+a commercial license. The ``conda-forge`` channel is free for all to use.
```
[the remark this answers is not in the document]
Comment (mattip) [reply docs/source/user-guide/concepts/channels.rst:23]: adopted

**Comment by mattip** [comment]
changes made. CI is waiting for a maintainer to approve running

**Review by chenghlee — APPROVED** [review]

File: docs/source/user-guide/tasks/manage-python.rst [hunk 4]
```diff
@@ -85,6 +85,16 @@ Python version into it:
 
       python --version
 
+Installing PyPy
+===============
+
+To use the PyPy builds you can do the following::
+
+    conda config conda config --add channels confa-forge
```
Comment (mattip) [inline docs/source/user-guide/tasks/manage-python.rst:93]: grr, typo. Will do another PR to fix it

**Pull request merged**